File: chatwoot/csharp/src/OSEG.ChatwootExamples/ConversationAddLabelsExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class ConversationAddLabelsExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("api_access_token", "USER_API_KEY");
        // config.ApiKey.Add("api_access_token", "AGENT_BOT_API_KEY");
        // config.ApiKey.Add("api_access_token", "PLATFORM_APP_API_KEY");

        var contactAddLabelsRequest = new ContactAddLabelsRequest(
            labels: [
            ]
        );

        try
        {
            var response = new ConversationLabelsApi(config).ConversationAddLabels(
                accountId: 0,
                conversationId: 0,
                data: contactAddLabelsRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling ConversationLabelsApi#ConversationAddLabels: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/typescript-axios/src/ConversationAddLabelsExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
  apiKey: "USER_API_KEY",
  // apiKey: "AGENT_BOT_API_KEY",
  // apiKey: "PLATFORM_APP_API_KEY",
});

const contactAddLabelsRequest: api.ContactAddLabelsRequest = {
  labels: [
  ],
};

new api.ConversationLabelsApi(configuration).conversationAddLabels(
  0, // accountId
  0, // conversationId
  contactAddLabelsRequest, // data
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling ConversationLabelsApi#conversationAddLabels:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
